test/libsolidity/ASTJSON/non_utf8.sol

```diff
@@ -1 +1,3 @@
 contract C { function f() public { var x = hex"ff"; } }
+
+// ----
```

Comment:
Are those delimiters also been made obligatory in ASTJSON tests with this PR?

Reply:
No, I think they were just added by the automatic expectation updater

Reply:
Ok!